index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="all-container">
+      <section class="background">
+        <header>
+          <nav class="navbar">
+            <a href="#" class="nav-branding">CESummit.</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </nav>
+        </header>
+        <article class="intro">
+          <h3 class="hellotxt">"Hello! Constructors"</h3>
+          <h1 class="creativetxt">
+            CONSTRUCTION ENGINEERING <br />
+            GLOBAL SUMMIT <br />
+            2020
+          </h1>
+          <p class="joyfultxt">
+            A Joyful celebration believing in the value of openness and sharing
+            creating a positive change with people from over 80 countries is
+            taking place in october, in Korea.
+          </p>
+          <p class="yeartxt">2015.10.15(THU) ~ 16(FRI)</p>
+          <p>@ National Museum of Korea, Art Center Nabi and more</p>
+        </article>
+      </section>
+
+      <article class="main mt-0 px-3 py-5">
+        <h3 class="main-program text-center text-white fw-bold">
```

Comment:
- [ ] As per the semantic HTML rules, kindly make sure to use h3 only if you already have h2 :recycle:

Reply:
Done